internal/seclang/directives.go

```diff
@@ -992,6 +992,15 @@ func directiveSecDataset(options *DirectiveOptions) error {
 	return nil
 }
 
+func directiveSecArgumentsLimit(options *DirectiveOptions) error {
```

Comment:
Please do add documentation as in https://github.com/corazawaf/coraza/pull/812/files#diff-ac03ba9c103ce13c145e2b00e0f7064decc5fdbaec3e05deafc491c54927d144L911-L927. You can use as reference https://github.com/SpiderLabs/ModSecurity/wiki/Reference-Manual-(v2.x)#secargumentslimit

Reply:
<3 Thanks!